cpg_workflows/jobs/mito.py

```diff
@@ -750,3 +752,54 @@ def parse_contamination_worker(
     )
 
     return j, contamination_level
+
+
+def mitoreport(
+    b: Batch,
+    sequencing_group: SequencingGroup,
+    vcf_path: Path,
+    cram_path: Path,
+    mito_ref: hb.ResourceGroup,
+    output_path: Path,
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    Run Mitoreport to generate html report of mito variants
+    """
+    job_attrs = job_attrs or {}
+    j = b.new_job('mitoreport', job_attrs)
+    j.image(image_path('mitoreport'))
+
+    res = STANDARD.request_resources(ncpu=2)
+    res.set_to_job(j)
```

Comment:
[`set_resources()`](https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/resources.py#L71) can do this in one line